app/store/services/helpers/gps.js

```diff
@@ -0,0 +1,51 @@
+const getDistance = (
+  pointA = { longitude: 0, latitude: 0 },
+  pointB = { longitude: 0, latitude: 0 }
+) => {
+  const R = 6371e3; // metres
+  const φ1 = pointA.latitude * Math.PI / 180;
```

Comment:
i like your notation but i am confused
you should use it the same way other people do

Reply:
i'm not the others! 🌵

Reply:
oui il meure a la fin

Reply:
no way!